roles/openshift_logging_fluentd/tasks/main.yaml

```diff
@@ -149,6 +149,21 @@
       - name: ops-cert
         path: "{{ openshift_logging_fluentd_ops_cert_path | default(generated_certs_dir ~ '/system.logging.fluentd.crt') }}"
 
+- name: Copy cluster-logging priority class definition
```

Comment:
At this time, since we are only using this for Fluentd I think this is the most appropriate place for it. If we were to utilize this in another logging component as well, then I think it would make the most sense to move it up to the `openshift_logging` role